src/components/Chart/parseChartData.js

```diff
@@ -12,8 +12,7 @@ export default function parseChartData(
 ) {
     const labels = []
     const datasets = []
-    const title = CATEGORIES[category].label
-    const subtitle = CATEGORIES[category].subtitle
+    const { label, subtitle } = CATEGORIES.find(c => c.value === category)
```

Comment:
The abstractions make it hard to judge whether this is always safe to do or not. Is it safe to rely on us always finding this category?

Reply:
Yes it is safe, essentially what we are doing here is iterating through the list of available options until we find the one with the selected value. Since the selected value is coming from the `onChange` callback of the `SelectField` it must be present in the available options. 

Would it help if I changed it a bit, like this:
```javascript
    // ...
    const selectedCategory = = CATEGORIES.find(c => c.value === category)
    // ...
    return {
        // ...
        title: selectedCategory.label,
        subtitle: selectedCategory.subtitle,
    }
```

Reply:
Thanks for explaining, I suspected that that was what was happening but wasn't sure. Ok, in that case I'll leave it to you 👍 Either syntax would work for me.

Reply:
I liked my suggestion above, see a73b52e